test/lit/passes/optimize-instructions-ignore-traps.wast

```diff
@@ -704,15 +704,7 @@
  ;; CHECK-NEXT:           (f32.const 71)
  ;; CHECK-NEXT:          )
  ;; CHECK-NEXT:         )
- ;; CHECK-NEXT:         (i64.eqz
- ;; CHECK-NEXT:          (select
- ;; CHECK-NEXT:           (i64.const 58)
- ;; CHECK-NEXT:           (i64.const -982757)
- ;; CHECK-NEXT:           (i64.eqz
- ;; CHECK-NEXT:            (i64.const 0)
```

Comment:
Please make this constant be a get of a local, to avoid large changes in this test.